modules/auxiliary/gather/impersonate_ssl.rb

```diff
@@ -180,6 +181,16 @@ def run
       ef.create_extension('subjectKeyIdentifier', 'hash'),
     ]
 
+    # add additional SAN entries to the new cert
+    if !datastore['ADD_SAN'].nil? && !datastore['ADD_SAN'].empty?
+      sans = datastore['ADD_SAN'].to_s.split(/,/)
+      sans.map! do |san|
+        san = san =~ Resolv::IPv4::Regex || san =~ Resolv::IPv6::Regex ? "IP:#{san}" : "DNS:#{san}"
```

Comment:
```suggestion
        san = (san =~ Resolv::IPv4::Regex || san =~ Resolv::IPv6::Regex) ? "IP:#{san}" : "DNS:#{san}"
```

Suggestion for making this a little easier to read.

Reply:
See https://support.f5.com/csp/article/K13471 for an example of how the IP and DNS fields might appear in a sample certificate. Adding this comment for future traveler's reference.